shared/helpers/apps.js

```diff
@@ -0,0 +1,19 @@
+const { hash: namehash } = require('eth-ens-namehash')
+
+const APPS = [
+  { id: 'agent', contractName: 'Agent' },
```

Comment:
It may be better calling the `id` `name` instead, or potentially changing it to be the `appName` (e.g. `agent.aragonpm.eth`) to be more consistent with the rest of the stack.